rpc/__tests__/event.ts

```diff
@@ -104,7 +104,7 @@ export const RPCEventTests = async function() {
                 });
                 fail('Did not Throw');
             } catch (err){
-                expect(err.message).toEqual('EventSubscription not found');
+                expect(err.message).toContain('EventSubscription not found');
```

Comment:
Does Jayson format its errors differently, or is this just a precautionary change?

Reply:
Error format still follows the JSON RPC 2 specification; however, the way the jayson errors are generated they included an outer context such as: `Invalid method parameter(s) [<detailed response>]`